### ISSUE TITLE
Euclid macros are not processed

### ISSUE THREAD

**Issue by jdavies-st** [mfouesneau/arxiv_on_deck_2#20]
The Euclid A&A template uses a `euclid.sty` file that defines a set of macros, and then the main.tex file references it via

```latex
\usepackage{euclid}
```

The most important of these is the `\Euclid` macro that renders _Euclid_.  Currently these are showing up in all Euclid paper titles as just `\Euclid`.

It would be good if .sty file could be pulled in and the `\newcommand` definitions used to interpret the main tex document.

**Comment by mfouesneau** [comment]
Yes, private packages are not handled. Packages may be very complex and interfere with finding the main elements in the document's core.

What happens if you replace \usepackage by \input?

**Comment by jdavies-st** [comment]
I will try that.  Thanks!

Yes, generally, these private packages can have all sorts of garbage in them, so generally inserting them is probably a bit dangerous.  I'll try with this euclid.sty file though and see what breaks.  =)

**Comment by mfouesneau** [comment]
Thanks. I suggest having the simple macros in a separate file from the package so they are safer to identify. That said, if Mathjax cannot parse it, there is not much we can do here.

In principle, the code finds any macro with 
`'providecommand', 'command', 'newcommand', 'renewcommand', 'def', 'gdef'`
